docs/rails.md

```diff
@@ -0,0 +1,35 @@
+# Rails conventions
+
+## App behavior should be configured via environment variables
+
+When implementing new features, it should not depend on current
+`RAILS_ENV` value, but environment variable(s) specific to the
+feature.
+
+* Avoid modifying `config/environments/*.rb`, and keep default
+  Rails configuration (this will also help when updating Rails);
```

Comment:
Just this tiny semi-colon to remove and we're good to go :+1:

Reply:
Done.